eth/backend.go

```diff
@@ -99,7 +102,7 @@ type Ethereum struct {
 
 // New creates a new Ethereum object (including the
 // initialisation of the common Ethereum object)
-func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
+func New(stack *node.Node, config *ethconfig.Config, l1Client sync_service.EthClient) (*Ethereum, error) {
```

Comment:
Should `EthClient` be a pointer so that you can pass in `nil`?

Reply:
It's an interface, it can be `nil`.